lib/bolt_server/transport_app.rb

```diff
@@ -30,6 +30,13 @@ def initialize(config)
       super(nil)
     end
 
+    def scrub_stack_trace(result)
```

Comment:
This name doesn't imply anything about jsonifying the result, I'd rather leave that separate.